src/app/utils/constants.ts

```diff
@@ -10,12 +10,15 @@ export enum NODE_TYPE {
   LIGHTNING_APP = 'LIGHTNING_APP',
   ZAP_DESKTOP = 'ZAP_DESKTOP',
   BTCPAY_SERVER = 'BTCPAY_SERVER',
+  LOOP = 'LIGHTNING LOOP',
 }
 
 export const DEFAULT_NODE_URLS = {
   [NODE_TYPE.LOCAL]: 'https://localhost:8080',
   [NODE_TYPE.LIGHTNING_APP]: 'https://localhost:8086',
   [NODE_TYPE.ZAP_DESKTOP]: 'https://localhost:8180',
+  // Need to update to https after testing
+  [NODE_TYPE.LOOP]: 'http://localhost:8081',
```

Comment:
I'm pretty sure Loop will normally run over HTTP since there're no credentials

Reply:
Ah shoot, yes that's correct. Removed that comment